packages/lit-dev-content/samples/base.json

```diff
@@ -3,6 +3,10 @@
     "package.json": {
       "content": "{\n  \"dependencies\": {\n    \"lit\": \"^2.0.0\",\n    \"@lit/reactive-element\": \"^1.0.0\",\n    \"lit-element\": \"^3.0.0\",\n    \"lit-html\": \"^2.0.0\"\n  }\n}",
       "hidden": true
+    },
+    "_check-code-helpers.js": {
```

Comment:
Should this only be included in tutorials with check-code enabled? I think currently this will appear in your shared gist.